src/search/fuse.js.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'fuse.js'
```

Comment:
I suggest moving this file up to the root of the project and renaming it `externals.d.ts` or `packages.d.ts` or similar.